db/migrate/20200625081552_add_unique_index_to_lookup_value.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToLookupValue < ActiveRecord::Migration[6.0]
+  def change
+    add_index :lookup_values, [:lookup_key_id, :match], unique: true
```

Comment:
Do we need any change in lookup_value.rb at validation defined on `match` for uniqueness?

Reply:
this key does exactly what the validation does, but on the database level, because we are hitting this issue: https://github.com/rails/rails/issues/20676

Reply:
ahh.. got it! Thank you @ezr-ondrej for giving a reference.